mixer/template/apikey/apiKey.pb.html

```diff
@@ -72,7 +72,7 @@ <h2 id="Template">Template</h2>
 </tr>
 <tr id="Template-timestamp">
 <td><code>timestamp</code></td>
-<td><code><a href="#istio-policy-v1beta1-TimeStamp">istio.policy.v1beta1.TimeStamp</a></code></td>
+<td><code><a href="https://istio.io/docs/reference/config/policy-and-telemetry/istio.policy.v1beta1.html#TimeStamp">istio.policy.v1beta1.TimeStamp</a></code></td>
```

Comment:
There are a lot of changes here from relative links to absolute links.  @geeknoid is this ok?

Reply:
Yeah, updated code gen. These types are now referenced instead of being inlined.